src/Microsoft.Azure.CosmosRepository.AspNetCore/Microsoft.Azure.CosmosRepository.AspNetCore.csproj

```diff
@@ -41,12 +41,12 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="3.1.22" />
-        <PackageReference Include="MinVer" Version="2.5.0">
+        <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="6.0.0" />
```

Comment:
Does this mean people using netcore3.1 would not be able to use the library? Not sure it's a huge issue, given that upgrade should have been done by now. But this could also pose a problem potentially for people running on net5?

Reply:
That's a good callout, I'm not sure. I'll have to ask around, but I was under the impression that since that package supports .NET Standard 2.0, it should work.

Reply:
I'm sure we had a issue raised about this in the past, I will try and dig it out.

Reply:
Did you hear anything back on this @IEvangelist ?

Reply:
Sorry, it's been all quiet on my end. I'll ask again. I'll let you know as soon as I can.

Reply:
I confirmed, that this package multi-targets and one of the TFMs is `netstandard2.0`. So this is fine.